src/components/AccountsList/index.js

```diff
@@ -36,8 +36,8 @@ function AccountsList({
   const [page, setPage] = useState(pageProp);
   const [pageSize, setPageSize] = useState(pageSizeProp);
   const fetcher = (url) => fetchJson(url);
-  const paginationString = new URLSearchParams({ page, pageSize }).toString();
-  const { data, mutate } = useSWR(apiUrl + paginationString, fetcher);
+  // const paginationString = new URLSearchParams({ page, pageSize }).toString();
```

Comment:
This was throwing an error after passing down the props, figured it was because of how the parameters were being passed down to useSWR, and also realized we do not have pagination for this page implemented, I asked @esirK  to look at it, and seems we might have a different data structure for this particular one - so in the meantime I have commented out that line 😅 @kilemensi 

<img width="1118" alt="Screenshot 2022-01-19 at 12 50 10" src="https://user-images.githubusercontent.com/12892109/150110103-29df4011-0809-4610-a428-c30b7f45d1f3.png">

Reply:
@gertie-sheshe we need to fix `pagination` on this page, either on this PR or another. This line isn't an issue at all!

We are passing undefined as page & pageProps ...so the string becomes `page=undefined&pageSize=undefined` ...and then in `useSWR` we are not concatenating the URL with `?` ...so the result URL becomes `/api/lists/{id}page=undefined&pageSize=undefined`

Anyways, if we want to hide Pagination for now, we should remove the entire
```
  <Pagination 
     ...
  />
```

Reply:
Yeah, I noticed that and it is fixed I just haven't pushed it because pagination hasn't been implemented on the backend (screenshot above), so even if I pass that it does not fix pagination. @esirK mentioned the structure will be different for this case so still waiting on feedback from him @KhadijaMahanga

Reply:
Removed Pagination, to be addressed after it is done on the backend as agreed after the discussion @KhadijaMahanga